data/test/scenarios/put_to_recall_and_modify.cfg

```diff
@@ -1,5 +1,9 @@
 #textdomain wesnoth-test
 
+# This is almost the same as the modify_unit_which_recall_list test, except that
+# creating Charlie on the map can leave an on-map x,y pair in Charlie's data,
+# which needs to be handled within the WML API.
```

Comment:
This no longer seems attached to the test that it's commenting on, I think it either needs to be accommodated in the new-style documentation or to be moved into the test's code.

Reply:
It's the only test in the file, so is there any confusion over which test it belongs to?

Reply:
Ah, I'd missed that detail. I agree, it's a lot clearer which test it belongs to in than case.